src/sentry_github/plugin.py

```diff
@@ -97,6 +112,39 @@ def get_allowed_assignees(self, request, group):
 
         return (('', 'Unassigned'),) + users
 
+    def get_allowed_issues(self, request, group):
+        issues = []
+        url = self.build_api_url(group, 'issues', query_params={'state': 'open', 'per_page': 100})
+        num_requests = 0
+        while url and num_requests < 10:
```

Comment:
probably need to figure out a good way to cache this since this could be multiple requests per form init... suggestions?